core/src/com/unciv/ui/worldscreen/unit/UnitActions.kt

```diff
@@ -462,22 +462,24 @@ object UnitActions {
 
     private fun addSpreadReligionActions(unit: MapUnit, actionList: ArrayList<UnitAction>, tile: TileInfo) {
         if (!unit.hasUnique("Can spread religion [] times")) return
-        if (unit.religion == null) return
+        if (unit.religion == null || unit.civInfo.gameInfo.religions[unit.religion]!!.isPantheon()) return
         val maxReligionSpreads = unit.maxReligionSpreads()
         if (!unit.abilityUsedCount.containsKey("Religion Spread")) return // This should be impossible anyways, but just in case
         if (maxReligionSpreads <= unit.abilityUsedCount["Religion Spread"]!!) return
         val city = tile.getCity()
+        if (city == null) return
         actionList += UnitAction(UnitActionType.SpreadReligion,
             title = "Spread [${unit.religion!!}]",
             action = {
                 unit.abilityUsedCount["Religion Spread"] = unit.abilityUsedCount["Religion Spread"]!! + 1
-                city!!.religion[unit.religion!!] = 100
+                city.religion.clearAllPressures()
```

Comment:
Radical! I would have expected such a "Spread Religion" to just add some points... And less than the points a holy city gets initially... (prejudice by name, not Civ5 knowledge)

Reply:
This is an interim, later on each city will have followers depending on how much the religion was spread to that city, and than spreading religion will only convert a few of these followers. That is yet to be implemented, so for now religions just replace each other